### DISCUSSION TITLE
If I want to inherit SingleFile to my safari, chrome extension, should I choose SingleFile or SingleFile Lite

### DISCUSSION THREAD

**Discussion by CaryTrivett** [gildas-lormeau/SingleFile#1125] [Q&A] [answered]
1. does SingleFile support v3?  If you want to support v3 , can you only use lite

2. Zotero connect and cite-extension both use SingleFile Lite, should I use Lite too?

**Comment by gildas-lormeau** [comment, marked as answer]
1. SingleFile Lite is the extension in MV3, it works but has some regressions (see "features" in the README). Note that both MV2 and MV3 relies on [single-file-core](https://github.com/gildas-lormeau/single-file-core) actually.
2. I wasn't aware they use the Lite version. I think you could use the Lite version. For the record, I do not publish it on the store because the auto-update is broken in MV3.
